### ISSUE TITLE
Adding Aarogya Setu website to we-are-using

### ISSUE THREAD

**Issue by vivek1996** [preactjs/preact-www#606]
Aarogya Setu is an Indian open-source cross-platform CoVID-19 "Contact tracing, Syndromic mapping and Self-assessment" digital service, primarily a mobile app, developed by the National Informatics Centre under the Ministry of Electronics and Information Technology India.
 
The website [https://www.aarogyasetu.gov.in/](https://www.aarogyasetu.gov.in/) is using Preact.
So, whether this will be a valid entry in we are using page?

**Comment by marvinhagemeister** [comment]
That's awesome! Yeah it's perfectly valid and we'd love to have it on that list :+1:

**Comment by Im-PJ** [comment]
> Aarogya Setu is an Indian open-source cross-platform CoVID-19 "Contact tracing, Syndromic mapping and Self-assessment" digital service, primarily a mobile app, developed by the National Informatics Centre under the Ministry of Electronics and Information Technology India.
> 
> The website https://www.aarogyasetu.gov.in/ is using Preact.
> So, whether this will be a valid entry in we are using page?

@vivek1996 Just curious to know the source of this information. Can you share some details.

**Comment by vivek1996** [comment]
> > Aarogya Setu is an Indian open-source cross-platform CoVID-19 "Contact tracing, Syndromic mapping and Self-assessment" digital service, primarily a mobile app, developed by the National Informatics Centre under the Ministry of Electronics and Information Technology India.
> > The website https://www.aarogyasetu.gov.in/ is using Preact.
> > So, whether this will be a valid entry in we are using page?
> 
> @vivek1996 Just curious to know the source of this information. Can you share some details.

Open the website and check the source maps

**Comment by bijay-ps** [comment]
> > > Aarogya Setu is an Indian open-source cross-platform CoVID-19 "Contact tracing, Syndromic mapping and Self-assessment" digital service, primarily a mobile app, developed by the National Informatics Centre under the Ministry of Electronics and Information Technology India.
> > > The website https://www.aarogyasetu.gov.in/ is using Preact.
> > > So, whether this will be a valid entry in we are using page?
> > 
> > 
> > @vivek1996 Just curious to know the source of this information. Can you share some details.
> 
> Open the website and check the source maps

I inspected the web page of https://aarogyasetu.gov.in/ in chrome and to me it looks like that it heavily using jquery. But I can't see anything related to preact.

**Comment by vivek1996** [comment]
> > > > Aarogya Setu is an Indian open-source cross-platform CoVID-19 "Contact tracing, Syndromic mapping and Self-assessment" digital service, primarily a mobile app, developed by the National Informatics Centre under the Ministry of Electronics and Information Technology India.
> > > > The website https://www.aarogyasetu.gov.in/ is using Preact.
> > > > So, whether this will be a valid entry in we are using page?
> > > 
> > > 
> > > @vivek1996 Just curious to know the source of this information. Can you share some details.
> > 
> > 
> > Open the website and check the source maps
> 
> I inspected the web page of https://aarogyasetu.gov.in/ in chrome and to me it looks like that it heavily using jquery. But I can't see anything related to preact.

Yeah... It seems the website has been updated entirely. Before it was just a static website using preact.

**Comment by Im-PJ** [comment]
Thanks for that update.  
I think Preact team should take a note of this as the site is no longer using preact library @marvinhagemeister

**Comment by marvinhagemeister** [comment]
Awesome, thank you so much to everyone for notifying us of the changes to their site! I've just removed it from the list :+1: :tada:
